### DISCUSSION TITLE
Not support for laravel 10

### DISCUSSION THREAD

**Discussion by alfirus** [jenssegers/laravel-mongodb#2534] [General]
Hi,

I install Laravel 10 and jenssegers/laravel-mongodb and it seem jenssegers not compatible. Any chance to upgrade?

**Comment by ATimrots** [comment]
Same for me. Waiting for Laravel 10 supported version.

**Comment by VincentVanWijk** [comment]
Same problem here, can't install this package on a fresh installation of Laravel 10

**Comment by abd-wazzan** [comment]
Same problem, I think the master branch [composer file](https://github.com/jenssegers/laravel-mongodb/blob/master/composer.json) is compatible with Laravel 10 but it was not released yet.

**Comment by zsinryu** [comment]
Can we add this support soon?

**Comment by alfinauzikri** [comment]
I just tried it and it is compatible with laravel 10

Install the package with the following command:
`composer require jenssegers/mongodb:dev-master`

![image](https://github.com/jenssegers/laravel-mongodb/assets/14070303/a2f2f088-5786-4f75-bf1c-e0c425427e63)

**Reply by zsinryu** [reply]
@alfinauzikri yes, but it is on dev-master only.

**Reply by alfinauzikri** [reply]
Laravel 10 support will be released when it is stable enough. As already discussed here: https://github.com/jenssegers/laravel-mongodb/issues/2500

**Comment by GromNaN** [comment]
I understand it's been a long wait, the release is planned for the next few weeks. https://github.com/mongodb-php/laravel-mongodb/issues/2499#issuecomment-1687960205
